systemtests/src/test/java/io/enmasse/systemtest/isolated/policy/NetworkPolicyTestBrokered.java

```diff
@@ -0,0 +1,255 @@
+import io.enmasse.admin.model.v1.BrokeredInfraConfig;
+import io.enmasse.admin.model.v1.BrokeredInfraConfigBuilder;
+import io.enmasse.admin.model.v1.BrokeredInfraConfigSpecAdminBuilder;
+import io.enmasse.admin.model.v1.BrokeredInfraConfigSpecBrokerBuilder;
+import io.enmasse.admin.model.v1.ResourceAllowance;
+import io.enmasse.admin.model.v1.ResourceRequest;
+import io.enmasse.admin.model.v1.StandardInfraConfig;
+import io.enmasse.admin.model.v1.StandardInfraConfigBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecAdminBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecBrokerBuilder;
+import io.enmasse.admin.model.v1.StandardInfraConfigSpecRouterBuilder;
+import io.enmasse.systemtest.Environment;
+import io.enmasse.systemtest.UserCredentials;
+import io.enmasse.systemtest.bases.TestBase;
+import io.enmasse.systemtest.bases.isolated.ITestIsolatedBrokered;
+import io.enmasse.systemtest.condition.OpenShift;
+import io.enmasse.systemtest.messagingclients.rhea.RheaClientReceiver;
+import io.enmasse.systemtest.messagingclients.rhea.RheaClientSender;
+import io.enmasse.systemtest.model.address.AddressType;
+import io.enmasse.systemtest.model.addressspace.AddressSpaceType;
+import io.enmasse.systemtest.platform.Kubernetes;
+import io.enmasse.systemtest.platform.apps.SystemtestsKubernetesApps;
+import io.enmasse.systemtest.utils.AddressUtils;
+import io.enmasse.systemtest.utils.PlanUtils;
+import io.enmasse.systemtest.utils.TestUtils;
+import io.fabric8.kubernetes.api.model.LabelSelector;
+import io.fabric8.kubernetes.api.model.LabelSelectorBuilder;
+import io.fabric8.kubernetes.api.model.PodTemplateSpec;
+import io.fabric8.kubernetes.api.model.networking.NetworkPolicyIngressRuleBuilder;
+import io.fabric8.kubernetes.api.model.networking.NetworkPolicyPeer;
+import io.fabric8.kubernetes.api.model.networking.NetworkPolicyPeerBuilder;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShift(version = 4)
+public class NetworkPolicyTestBrokered extends TestBase implements ITestIsolatedBrokered {
+    private UserCredentials credentials = new UserCredentials("test", "test");
+    private String blockedSpace = "blocked-namespace";
+    private String allowedSpace = "allowed-namespace";
+
+    @AfterEach
+    void clearNamespaces() throws Exception {
+        kubernetes.deleteNamespace(blockedSpace);
+        TestUtils.waitForNamespaceDeleted(kubernetes, blockedSpace);
+        kubernetes.deleteNamespace(allowedSpace);
+        TestUtils.waitForNamespaceDeleted(kubernetes, allowedSpace);
+    }
+
+    @Test
+    void networkPolicyTestWithPodSelector() throws Exception {
```

Comment:
Be consistent and use the testFooBar format for test methods rather than starting a new trend :)